tests/torch/graph/test_graph_pattern.py

```diff
@@ -0,0 +1,108 @@
+from nncf.torch.graph.patterns import GraphPattern
+import networkx as nx
+
+import itertools
+
+
+def test_ops_combination_two_patterns():
+    first_type = ['a', 'b']
+    second_type = ['c', 'd']
+
+    first_pattern = GraphPattern(first_type)
+    second_pattern = GraphPattern(second_type)
+
+    ref_pattern = GraphPattern(first_type)
```

Comment:
The pattern matching tests should also be made common if possible.

Reply:
Done